core/src/main/scala/org/apache/spark/deploy/SparkSubmit.scala

```diff
@@ -904,7 +904,10 @@ private[spark] class SparkSubmit extends Logging {
       childArgs ++= Seq("--verbose")
     }
 
-    sparkConf.set("spark.app.submitTime", System.currentTimeMillis().toString)
+    if (!isKubernetesClusterModeDriver) {
```

Comment:
Simply setIfMissing？

Reply:
> Simply setIfMissing？ then it will be set when we first touch it

If user provide conf `spark.app.submitTime`, I think we should overwrite it.

Reply:
I am neutral about `override it`